app/assets/stylesheets/spotlight/_autocomplete.scss

```diff
@@ -0,0 +1,53 @@
+auto-complete {
+  width: 100%;
+  z-index: $zindex-dropdown !important; /* get typeahead to float on top of e.g. leaflet */
+  ul {
+    list-style-type: none;
+    z-index: $zindex-dropdown !important;
+    margin-top: 2px;
+    max-height: 360px;
+    overflow-y: auto;
+    padding: 8px 0;
+    background-color: #fff;
+    border: 1px solid #ccc;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+    -webkit-border-radius: 8px;
```

Comment:
Is there a reason we need these to have the browser prefixes?

Reply:
No. I've removed them.